docs/content/en/docs/operator-manual/piped/installation/installing-on-fargate.md

```diff
@@ -0,0 +1,149 @@
+  #     publicKeyData: {BASE64_ENCODED_PUBLIC_KEY}
+```
+
+- Create a new secret in [SecretManager](https://aws.amazon.com/secrets-manager/) to store above configuration data securely
+
+```console
+aws secretsmanager create-secret --name PipedConfig \
+  --description "Configuration of piped running as ECS Fargate task" \
+  --secret-string file://piped-config.yaml
+```
+
+Note: Make sure your account has permission to execute CreateSecret operation or you may get `AccessDeniedException` error from AWS Api. In that case, you can try creating secret via AWS Web console.
+
+- Prepare task definition for your piped task. Basically, you can just define your piped TaskDefinition as normal TaskDefinition, the only thing that needs to be beware is, to enable your piped accesses it's configuration we created as a secret on above, you need to add `secretsmanager:GetSecretValue` policy to your piped task `executionRole`. Read more in [Required IAM permissions for Amazon ECS secrets](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/specifying-sensitive-data-secrets.html).
+
+A sample TaskDefinition for Piped as following.
+
+```json
+{
+  "family": "piped",
+  "executionRoleArn": "{PIPED_TASK_EXECUTION_ROLE_ARN}",
+  "containerDefinitions": [
+    {
+      "name": "piped",
+      "image": "gcr.io/pipecd/piped:{{< blocks/latest_version >}}",
+      "command": [
+        "echo $configData > /etc/piped/config.yaml; piped --config-file=/etc/piped/config.yaml --metrics=true"
+      ],
+      "secrets": [
+        {
+          "valueFrom": "{PIPED_SECRET_MANAGER_ARN}",
+          "name": "configData"
+        }
+      ]
+    }
+  ],
+  "compatibilities": [
+    "FARGATE"
+  ],
+  "requiresCompatibilities": [
+    "FARGATE"
+  ],
+  "networkMode": "awsvpc",
+  "memory": 512,
+  "cpu": 256,
+}
+```
+
+Register this piped task definition and start pipe task:
+
+```console
+aws ecs register-task-definition --cli-input-json file://taskdef.json
+aws ecs run-task --cluster {ECS_CLUSTER} --task-definition piped
+```
+
+Once the task is created, it will run continuously because of the piped execution. Since this task is run without [startedBy](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_StartTask.html#API_StartTask_RequestSyntax) setting, in case the piped is stopped, it will not automatically be restarted. To do so, you must define an ECS service to control piped task deployment.
+
+A sample Service definition to control piped task deployment.
+
+```json
+{
+  "cluster": "{ECS_CLUSTER}",
+  "serviceName": "piped",
+  "desiredCount": 1, # This must be 1.
+  "taskDefinition": "{PIPED_TASK_DEFINITION_ARN}",
+  "deploymentConfiguration": {
+    "maximumPercent": 100,
+    "minimumHealthyPercent": 100
+  },
+  "schedulingStrategy": "REPLICA",
+  "launchType": "FARGATE",
+  "networkConfiguration": {
+    "awsvpcConfiguration": {
+      "assignPublicIp": "ENABLED", # This is need to enable ECS deployment pull piped container image.
```

Comment:
JSON doesn't support any kind of comment, but so what!
```suggestion
      "assignPublicIp": "ENABLED", # This is need to enable ECS deployment to pull piped container images.
```

Reply:
Yes, it's. This code block comments are for explanation and we can expect users to understand that Json's comments are not allowed, as you said 😄